docker-compose.yml

```diff
@@ -2,18 +2,32 @@ version: '2'
 
 services:
   db:
+    restart: 'always'
     image: 'postgres:9.5'
     ports:
       - '5432:5432'
     volumes:
       - 'db:/var/lib/postgresql/data'
+
   app:
+    restart: 'always'
     depends_on:
       - 'db'
     build: .
     ports:
       - '3000:3000'
-    env_file: '.env'
+    volumes:
+      - '.:/test_track'
+    volumes_from:
+      - bundler_cache
+    command: ./script/serve
+    environment:
+      - DATABASE_URL=postgresql://postgres@db/test_track_development
```

Comment:
Yeah, at the moment. I figure that if someone wants to run TT in production then they'll probably either fork the repo or just run the TT container outside of compose.